src/conf/experiment/defaults.yaml

```
@@ -2,4 +2,5 @@
 experiment:
   debug: False
   manual_seed: 1234
-  name: null
\ No newline at end of file
+  name: null
+  bucket_name: null
```

Why not put our s3://deepvoice-experiments/ bucket as ... well, defaults? :D
```suggestion
  bucket_name: 's3://deepvoice-experiments/ '
```

---

so it won't break for non-deepvoice users.